lib/openstax/book.rb

```diff
@@ -0,0 +1,44 @@
+require_relative './page'
+
+module Openstax
+  class Book
+    attr_reader :uuid, :version
+
+    def initialize(uuid:, version:)
+      @uuid = uuid
+      @version = version
+    end
+
+    def data
+      @data ||= begin
+        book_json_url = "https://archive.cnx.org/contents/#{uuid}@#{version}.json"
+
+        uri = URI(book_json_url)
+        response = Net::HTTP.get(uri)
+        JSON.parse(response)
+      end
+    end
+
+    def pages
```

Comment:
We will need to handle units in addition to chapters.  Like what goes on here https://github.com/openstax/tutor-server/blob/master/lib/openstax/cnx/v1/book_part.rb#L32-L40 -- book parts can contain other book parts or just pages.

Reply:
@jpslav i added a recursive fetch of the page_ids from the TOC.  please check it out to see if it looks correct.  i tried it on the anatomy book....